packages/generator-electrode/component/templates/packages/component/_package.json

```diff
@@ -28,7 +28,9 @@
   "scripts": {
     "prepublish": "gulp prepublish",
     "start": "gulp demo",
-    "test": "gulp check"
+    "test": "gulp check",
+    "format": "prettier<% if (quoteType === "'") { %> --single-quote<% } %> --write \"{src,test}/**/*.{js,jsx}\"",
```

Comment:
@didi0613 fixed in https://github.com/electrode-io/electrode/pull/351/commits/39d4a89df5507d9af1d6829192e87f401996c614